contracts/src/libraries/access_control.cairo

```diff
@@ -1,42 +1,66 @@
-#[contract]
+use starknet::ContractAddress;
+#[starknet::interface]
+trait IAccessController<TContractState> {
```

Comment:
Maybe makes more sense under the access_controller module rather than in libraries